src/main/kotlin/org/digma/intellij/plugin/ui/common/Panels.kt

```diff
@@ -88,3 +96,33 @@ fun createTopPanel(project: Project, model: PanelModel): DigmaResettablePanel {
 fun wrapWithNoConnectionWrapper(project: Project, panel: DigmaTabPanel): DigmaTabPanel {
     return NoConnectionWrapper(project, panel)
 }
+
+private fun getGeneralRefreshButton(project: Project): JButton {
+    val insightsActionListener: InsightsViewService = project.getService(InsightsViewService::class.java)
+    val errorsActionListener: ErrorsViewService = project.getService(ErrorsViewService::class.java)
+
+    val size = Laf.scalePanels(Laf.Sizes.BUTTON_SIZE_24)
+    val buttonsSize = Dimension(size + 2, size + 2)
+    val generalRefreshIconButton = IconButton(Laf.Icons.Insight.REFRESH)
+    generalRefreshIconButton.preferredSize = buttonsSize
+    generalRefreshIconButton.maximumSize = buttonsSize
+    generalRefreshIconButton.toolTipText = asHtml(REFRESH_ALL_INSIGHTS_AND_ERRORS)
+
+    generalRefreshIconButton.addMouseListener(object : MouseAdapter() {
+        override fun mouseClicked(e: MouseEvent?) {
+            insightsActionListener.refreshInsights()
+            errorsActionListener.refreshErrors()
+        }
+    })
+    return generalRefreshIconButton
+}
+
+private fun getScopeLineResultPanel(scopeLine: DialogPanel, project: Project): JPanel {
+    val scopeLineResultPanel = JPanel()
+    scopeLineResultPanel.layout = BoxLayout(scopeLineResultPanel, BoxLayout.LINE_AXIS)
+    scopeLineResultPanel.add(Box.createHorizontalGlue())
```

Comment:
What is the purpose of the first `Box.createHorizontalGlue()` (I understand what the second one does)?